Tests/ApacheBeamTests/Pipeline/FileIOTests.swift

```diff
@@ -24,15 +24,19 @@
 //
 import ApacheBeam
 import Logging
-import XCTest
+import Testing
+import Foundation
 
-final class FileIOTests: XCTestCase {
-    override func setUpWithError() throws {}
-
-    override func tearDownWithError() throws {}
+final class FileIOTests {
 
+//    let containers: TestContainers
```

Comment:
Either clean up the commented out bits, or add comments justifying keeping them.